### DISCUSSION TITLE
“apple_cloud_notes_parser” No such file or directory -- notes_cloud_ripper.rb (LoadError)

### DISCUSSION THREAD

**Discussion by JayDunkin** [community/community#43535] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

So I’ve (I think) downloaded you Ruby project “[apple_cloud_notes_parser](https://github.com/threeplanetssoftware/apple_cloud_notes_parser)”, Ruby, Homebrew and Asdf. Followed the install instructions provided but am receiving an error message that it can’t find "notes_cloud_ripper.rb “ file.

**Comment by jfullstackdev** [comment]
seems that a specific project hosted on GitHub, go there at their Issues tab and ask the same question

**Reply by JayDunkin** [reply]
cheers
